src/jdk.jpackage/share/classes/jdk/jpackage/internal/resources/HelpResources_zh_CN.properties

```diff
@@ -34,7 +34,7 @@ MSG_Help_win_install=\  --win-dir-chooser\n          添加一个对话框以允
 MSG_Help_mac_launcher=\  --mac-package-identifier <ID string>\n          用来唯一地标识 macOS 应用程序的标识符\n          默认为主类名称。\n          只能使用字母数字 (A-Z,a-z,0-9)、连字符 (-) 和\n          句点 (.) 字符。\n  --mac-package-name <name string>\n          出现在菜单栏中的应用程序名称\n          这可以与应用程序名称不同。\n          此名称的长度必须小于 16 个字符，适合\n          显示在菜单栏中和应用程序“信息”窗口中。\n          默认为应用程序名称。\n  --mac-package-signing-prefix <prefix string>\n          在对应用程序包签名时，会在所有需要签名\n          但当前没有程序包标识符的组件的\n          前面加上此值。\n  --mac-sign\n          请求对程序包或预定义的应用程序映像\n          进行签名。\n  --mac-signing-keychain <keychain name>\n          要用来搜索签名身份的密钥链的名称\n          如果未指定，则使用标准的密钥链。\n  --mac-signing-key-user-name <team name>\n          Apple 签名身份的团队或用户名称部分。为了直接\n          控制用于对应用程序映像或安装程序进行签名的\n          签名身份，请使用 --mac-app-image-sign-identity 和/或\n          --mac-installer-sign-identity。此选项不能与\n          --mac-app-image-sign-identity 或 --mac-installer-sign-identity 结合使用。\n  --mac-app-image-sign-identity <identity>\n          用于对应用程序映像进行签名的身份。此值将直接\n          传递至 "codesign" 工具的 --sign 选项。此选项不能\n          与 --mac-signing-key-user-name 结合使用。\n  --mac-installer-sign-identity <identity>\n          用于对 "pkg" 安装程序进行签名的身份。此值将直接\n          传递至 "productbuild" 工具的 --sign 选项。此选项不能\n          与 --mac-signing-key-user-name 结合使用。\n  --mac-app-store\n          指示 jpackage 输出面向\n          Mac App Store。\n  --mac-entitlements <file path>\n          包含一些权利的文件的路径，在对捆绑包中的可执行文件\n          和库进行签名时会使用这些权利。\n  --mac-app-category <category string>\n          用于构造应用程序 plist 中 LSApplicationCategoryType 的\n          字符串。默认值为 "utilities"。\n
-MSG_Help_linux_install=\  --linux-package-name <package name>\n          Linux 程序包的名称，默认为应用程序名称\n  --linux-deb-maintainer <email address>\n          .deb 程序包的维护程序\n  --linux-menu-group <menu-group-name>\n          此应用程序所在的菜单组\n  --linux-package-deps\n          应用程序所需的程序包或功能\n  --linux-rpm-license-type <type string>\n          许可证的类型（RPM .spec 的 "License: <value>"）\n  --linux-app-release <release value>\n          RPM <name>.spec 文件的发行版值或 \n          DEB 控制文件的 Debian 修订版值\n  --linux-app-category <category value>\n          RPM <name>.spec 文件的组值或 \n          DEB 控制文件的节值\n  --linux-shortcut\n          为应用程序创建快捷方式。\n
+MSG_Help_linux_install=\  --linux-package-name <package name>\n          Linux 程序包的名称，默认为应用程序名称\n  --linux-deb-maintainer <email address>\n          .deb 程序包的维护程序\n  --linux-menu-group <menu-group-name>\n          此应用程序所在的菜单组\n  --linux-package-deps <package-dep>[, <package-dep>...]\n          应用程序所需的程序包或功能\n  --linux-rpm-license-type <type string>\n          许可证的类型（RPM .spec 的 "License: <value>"）\n  --linux-app-release <release value>\n          RPM <name>.spec 文件的发行版值或 \n          DEB 控制文件的 Debian 修订版值\n  --linux-app-category <category value>\n          RPM <name>.spec 文件的组值或 \n          DEB 控制文件的节值\n  --linux-shortcut\n          为应用程序创建快捷方式。\n
```

Comment:
`--linux-package-deps <package-dep-string>` to sync with HelpResources.properties